components/gitpod-cli/cmd/rebuild.go

```diff
@@ -124,10 +124,12 @@ func runRebuild(ctx context.Context, supervisorClient *supervisor.SupervisorClie
 		fmt.Println("")
 		fmt.Println("https://www.gitpod.io/docs/configure/workspaces/workspace-image#use-a-public-docker-image")
 		event.Set("ErrorCode", utils.RebuildErrorCode_NoCustomImage)
-		return err
+		return nil
 	}
 
-	err = os.WriteFile(filepath.Join(tmpDir, "Dockerfile"), []byte(baseimage), 0644)
+	tmpDockerfile := filepath.Join(tmpDir, "Dockerfile")
```

Comment:
I would prefer to keep in tmp folder. We should also make sure that it container stops, a user closes VS Code terminal or simply interrupts then this file should not be preserved